src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -1079,6 +1152,60 @@ class spell_dru_typhoon : public SpellScriptLoader
         }
 };
 
+// 67353 - T9 Feral Relic (Idol of Mutilation)
+class spell_dru_t9_feral_relic : public SpellScriptLoader
+{
+public:
+    spell_dru_t9_feral_relic() : SpellScriptLoader("spell_dru_t9_feral_relic") { }
+
+    class spell_dru_t9_feral_relic_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_dru_t9_feral_relic_AuraScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(SPELL_DRUID_T9_FERAL_RELIC_BEAR)
+                || !sSpellMgr->GetSpellInfo(SPELL_DRUID_T9_FERAL_RELIC_CAT))
+                return false;
+            return true;
+        }
+
+        void HandleProc(AuraEffect const* aurEff, ProcEventInfo& eventInfo)
+        {
+            PreventDefaultAction();
+            if (Unit* target = eventInfo.GetActionTarget())
+            {
+                uint32 triggerspell = 0;
+
+                switch (target->GetShapeshiftForm())
+                {
+                    case FORM_BEAR:
+                    case FORM_DIREBEAR:
+                        triggerspell = SPELL_DRUID_T9_FERAL_RELIC_BEAR;
+                        break;
+                    case FORM_CAT:
+                        triggerspell = SPELL_DRUID_T9_FERAL_RELIC_CAT;
+                        break;
+                    default:
+                        return;
+                }
+
+                target->CastSpell(target, triggerspell, true, nullptr, aurEff);
```

Comment:
Default cast call in Unit::HandleProcTriggerSpell was ```actor->CastSpell(actionTarget, ...``` unless overriden like this https://github.com/TrinityCore/TrinityCore/pull/17122/files#diff-4522eeffcbc961602e86620011a2d579L7941
So this here isn't correct